boot/bootutil/src/encrypted.c

```diff
@@ -766,25 +786,7 @@ boot_encrypt(struct enc_key_data *enc_state, int image_index,
 
     enc = &enc_state[rc];
     assert(enc->valid == 1);
-    for (i = 0; i < sz; i++) {
```

Comment:
This block was simplified since it seemed to reimplement AES CTR mode by hand.

Reply:
Ah, that explains it. I was wondering about it but couldn't find any comments about it.